src/UI/class-admin-columns-parsely-stats.php

```diff
@@ -290,8 +289,7 @@ public function get_parsely_stats_response( $analytics_api ) {
 
 		$response = $analytics_api->get_posts_analytics(
 			array(
-				'period_start'   => get_utc_date_format( - Analytics_Posts_API::ANALYTICS_API_DAYS_LIMIT ),
-				'period_end'     => get_utc_date_format(),
+				'period_start'   => Analytics_Posts_API::ANALYTICS_API_DAYS_LIMIT . 'd',
```

Comment:
`period_end` is by default current time so therefore just passing `period_start`